includes/admin.form.inc

```diff
@@ -186,25 +190,10 @@ function islandora_paged_content_admin_settings_form(array $form, array &$form_s
  *   A message in html detailing if the given executable is accessible.
  */
 function islandora_paged_content_admin_settings_form_executable_available_message($path, $version = NULL, $required_version = NULL) {
-  $available = is_executable($path);
-  if ($available) {
-    $image = theme_image(array('path' => 'misc/watchdog-ok.png', 'attributes' => array()));
-    $message = t('Executable found at @path', array('@path' => $path));
-    if ($version) {
-      $message .= t('<br/>Version: @version', array('@version' => $version));
-    }
-    if ($required_version) {
-      $message .= t('<br/>Required Version: @version', array('@version' => $required_version));
-      if (version_compare($version, $required_version) < 0) {
-        $image = theme_image(array('path' => 'misc/watchdog-error.png', 'attributes' => array()));
-      }
-    }
-  }
-  else {
-    $image = theme_image(array('path' => 'misc/watchdog-error.png', 'attributes' => array()));
-    $message = t('Unable to locate executable at @path', array('@path' => $path));
-  }
-  return $image . $message;
+  $message = islandora_deprecated('7.x-1.13', t('Please use islandora_executable_available_message() instead.'));
```

Comment:
Looks like this would need a `module_load_include('inc', 'islandora', 'includes/utilities');`, to make sure both of the functions called are available.